bundle.yaml

```diff
@@ -5,14 +5,14 @@
 # deployment before moving to production.
 #
 series: xenial
-applications:
+services:
```

Comment:
The `applications' key is a bundle v4 convention. While I am +1 to this change, it makes me sad that our testing tools do not support what is exported from the Juju GUI.